packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1482,11 +1495,15 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   @override
   Widget build(BuildContext context) {
     assert(debugCheckHasOverlay(context));
+    final SelectionRegistrar? registrar = SelectionContainer.maybeOf(context);
     Widget result = SelectionContainer(
-      registrar: this,
+      registrar: registrar ?? this,
```

Comment:
I think this is a change in functionality as now users cannot constrain a selection to a `SelectionArea`. Previously you cannot select across `SelectionArea`s. Why is this change needed for the copy interceptor?

Reply:
This behavior has been removed in favor of using `SelectionContainer` or a wrapper widget